src/components/Search.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../styles/Search.css';
+import getImages from '../requests/getImages';
+
+const Search = ({ setSearchResults, searchText, setSearchText }) => {  
+
+    const handleSubmit = async (event) => {
+        event.preventDefault();
+        setSearchResults(await getImages(searchText));
+    };
+
+    return (
+        <>
```

Comment:
don't need this fragment <> if you have a form already surrounding all your other tags. You just need one returned tag as the parent of all children in your return statement.